.github/actions/e2e/action.yaml

```diff
@@ -0,0 +1,44 @@
+name: 'Kuma E2E'
+description: 'Run end to end tests for kuma'
+inputs:
+  k8sVersion:
+    description: version of k3s to use or "kind" and "kindIpv6"
+    default: v1.28.1-k3s1
+    required: false
+  parallelism:
+    description: level of parallelization
+    default: '1'
+    required: false
+  target:
+    description: makefile target without e2e prefix
+    default: ""
+    required: false
+  arch:
+    description: The golang arch
+    default: amd64
+    required: false
+  cniNetworkPlugin:
+    description: The CNI networking plugin to use [flannel | calico]
+    default: flannel
+    required: false
+  deltaKDS:
+    description: if should run tests with new implementation of KDS
+    default: 'false'
+    required: false
```

Comment:
I think we can likely drop this option in the matrix

Reply:
What about just let it be here for now, since we are not actually running anything.
And I wrote the wrong name here, shoudl be `legacyKDS`, I'll correct it.

Reply:
Do you want to remove the `legacyKDS` stuff? If yes, let me do it in the second PR.